src/game/map.rs

```diff
@@ -58,10 +63,52 @@ pub fn get_world_size() -> u32 {
 /// See [http://docs.screeps.com/api/#Game.map.isRoomAvailable]
 ///
 /// [http://docs.screeps.com/api/#Game.map.isRoomAvailable]: http://docs.screeps.com/api/#Game.map.isRoomAvailable
+#[deprecated(
```

Comment:
No need for that. The point of being in ver <1.0 is to savagely break the API every 0.x update.

Reply:
Ah, the changes are in PTR... Then I'm not even sure whether we should deprecate it right now. Sometimes, features spend months in PTR before getting released.